draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -7868,6 +7863,45 @@ Content-Range: bytes 734-1233/1234
 </ul>
 </section>
 
+<section title="Partial PUT" anchor="partial-PUT">
+  <x:anchor-alias value="partial PUT"/>
+  <iref primary="false" item="Fields" subitem="Content-Range" x:for-anchor=""/><iref primary="false" item="Header Fields" subitem="Content-Range" x:for-anchor=""/><iref primary="false" item="Content-Range header field" x:for-anchor=""/>
+<t>
+   Some origin servers support <x:ref>PUT</x:ref> of a partial representation
+   when a <x:ref>Content-Range</x:ref> header field
+   (<xref target="field.content-range"/>) is sent in the request, though
+   such support is inconsistent and depends on private agreements with
+   user agents. In general, it requests that the state of the
+   <x:ref>target resource</x:ref> be partly replaced with the enclosed payload
+   data at an offset and length indicated by the Content-Range value.
+</t>
+<t>
+   An origin server &SHOULD; respond with a <x:ref>400 (Bad Request)</x:ref>
+   status code if it receives <x:ref>Content-Range</x:ref> on a PUT for a
+   target resource that does not support partial PUT requests.
+</t>
+<t>
+   If an origin server supports <x:ref>Content-Range</x:ref> on PUT but the
+   range provided is inconsistent or invalid for that resource (as defined by
+   the origin server), the origin server &SHOULD; respond with either the
+   <x:ref>412 (Precondition Failed)</x:ref> status code (if an applicable
+   precondition is false) or the <x:ref>409 (Conflict)</x:ref> status code
+   (if not).
+</t>
```

Comment:
Remove this bit as well because it assumes too much.
```suggestion
<t>
   If an origin server supports <x:ref>Content-Range</x:ref> on PUT but the
   range provided is inconsistent or invalid for that resource (as defined by
   the origin server), the origin server &SHOULD; respond with either the
   <x:ref>412 (Precondition Failed)</x:ref> status code (if an applicable
   precondition is false) or the <x:ref>409 (Conflict)</x:ref> status code
   (if not).
</t>
```